src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shp/LocalServerDynamicWorkspaceShapefileSample.java

```diff
@@ -0,0 +1,172 @@
+package com.esri.samples.localserver.local_server_dynamic_workspace_shp;
```

Comment:
package name has to be the same as the sample: `local_server_dynamic_workspace_shapefile`.